scripts/decode-syseeprom

```diff
@@ -109,6 +122,9 @@ def run(target, opts, args):
         pass
 
     if opts.init:
+        err = target.update_eeprom_db(e)
+        if err:
+            print "Failed to update eeprom database"
```

Comment:
Same as above.

Reply:
update_eeprom_db() will return 0 for success and -1 for error.